src/filters/types.hpp

```diff
@@ -55,6 +55,11 @@ namespace dmxfish::filters {
         filter_lua_script = 50,
         filter_8bit_to_float = 51,
         filter_16bit_to_float = 52,
-        filter_pixel_to_floats = 53
+        filter_pixel_to_floats = 53,
+        filter_float_map_range_8bit = 54,
+        filter_float_map_range_16bit = 55,
+        filter_float_map_range_float = 56,
+        filter_dual_byte_to_16bit = 57,
+        filter_one_byte_to_16bit = 58
```

Comment:
I would choose a name that makes sure that people know that it shifts the byte to the upper one.

Reply:
for the dual byte or the single one? i dont have an good idea..
for the dual byte: filter_dual_byte_merge ?
for the one byte: filter_map_8bit_to_16bit ?

Reply:
Your suggested name for the single byte one sounds good. For the dual byte one I would suggest `combine_bytes_to_16bit`.